src/mistral_common/tokens/tokenizers/tekken.py

```diff
@@ -283,6 +294,19 @@ def image(self) -> Optional[ImageConfig]:
     def image(self, value: ImageConfig) -> None:
         raise ValueError("Can only set Image config at init")
 
+    @property
+    def audio(self) -> Optional[AudioConfig]:
+        r"""The audio configuration of the tokenizer.
+
+        Returns:
+            Optional[AudioConfig]: The audio configuration object if it exists, otherwise None.
+        """
```

Comment:
```suggestion
    def audio(self) -> Optional[AudioConfig]:
        r"""The audio configuration of the tokenizer.

        Returns:
             The audio configuration object if it exists, otherwise None.
        """
```